Formula/dar.rb

```diff
@@ -10,6 +10,19 @@ class Dar < Formula
     sha256 "a74696553c86f94dfd1ebdad29ad9c5d22c79d78fc023e43bb86f272fbfc4d87" => :mavericks
   end
 
+  devel do
+    url "https://downloads.sourceforge.net/project/dar/dar/2.5.5/dar-2.5.5.tar.gz"
+    sha256 "aa77d6778206b00ad7b37d5f615aafff3ea770b35d96625429c14ff67b78466d"
+
+    depends_on "gcc" if MacOS.version <= :el_capitan
```

Comment:
The linkage is only a consequence of test-bot, which  behavior which rescues the fails_with exception and chooses to `brew install gcc` as its workaround. So this isn't really legitimate unless we know we're dealing with the bottled version, which certainly isn't true (yet) for devel specs and is also not really true for stable specs with bottles either.

So this is mainly a workaround for the cases where test-bot fails to install gcc before running brew test on a gcc linked formula. That might be fixable by checking brew linkage for gcc and brew installing gcc before running brew test if gcc isn't already installed. Alternatively we could rescue the test failure and install gcc at that point if the error was a failure to run due to a missing gcc library,  then call `retry`, else call `raise` for other any other exception (which is basically the same thing test-bot is already doing for installs) but the end result should be the same either way, where the chosen medicine is preventive or curative.